src/main/java/io/anserini/search/RetrieveMain.java

```diff
@@ -0,0 +1,71 @@
+package io.anserini.search;
+
+import org.apache.commons.io.FileUtils;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.OptionHandlerFilter;
+import org.kohsuke.args4j.ParserProperties;
+
+import java.io.File;
+import java.util.List;
+
+/*
+ * Java rewrite of retrieve.py
+ */
+public class RetrieveMain {
+    public static void main(String[] args) throws Exception {
+        SearchArgs searchArgs = new SearchArgs();
```

Comment:
Create a new args object, since this is completely different code path and most of the `SearchArgs` fields aren't relevant.

Reply:
Line up the args to be the same as the python script.